dockerfiles/benchmarks/README.md

```diff
@@ -50,25 +268,26 @@ The script sends metrics to Victoria Metrics.
 ci example:
 
 ```yml
-
 send-becnh-result:
   stage:                           send-becnh-result
   image:                           paritytech/benchmarks:latest
   variables:
     PROMETHEUS_URL:                "http://vm-longterm.parity-build.parity.io"
+    PROMETHEUS_URL:                "http://pushgateway.parity-build.parity.io"
```

Comment:
I assume the vm-longterm line above should be deleted?

And we might as well fix the "becnh" typo in the job name while we're editing this :)

Reply:
Oh, and it looks like the pushgateway is also available on https

Reply:
> I assume the vm-longterm line above should be deleted?
> 
> And we might as well fix the "becnh" typo in the job name while we're editing this :)

I left it there intentionally to reflect the history, but maybe it's a better idea to remove it

Reply:
> Oh, and it looks like the pushgateway is also available on https

good point, I will add it

Reply:
> I left it there intentionally to reflect the history, but maybe it's a better idea to remove it

Yeah I think leaving the old URL definition would just be confusing, given the new one will override it anyway.